tools/setup_helpers/env.py

```diff
@@ -45,7 +45,7 @@ def hotpatch_var(var, prefix='USE_'):
 def hotpatch_build_env_vars():
     # Before we run the setup_helpers, let's look for NO_* and WITH_*
     # variables and hotpatch environment with the USE_* equivalent
-    use_env_vars = ['CUDA', 'CUDNN', 'FBGEMM', 'MIOPEN', 'MKLDNN', 'NNPACK', 'DISTRIBUTED',
+    use_env_vars = ['CUDA', 'CUDNN', 'FBGEMM', 'MIOPEN', 'MKLDNN', 'MKLDNN_CBLAS', 'NNPACK', 'DISTRIBUTED',
```

Comment:
You do not need to change this line. `USE_*` variables are already automatically passed in tools/setup_helpers/cmake.py and this hotpatch function is likely gonna be removed soon.

Reply:
Thanks very much for your reminder. Have fixed it now.